frontend/src/metabase/parameters/components/FormattedParameterValue/FormattedParameterValue.tsx

```diff
@@ -20,22 +28,76 @@ function FormattedParameterValue({
   value,
   placeholder,
 }: FormattedParameterValueProps) {
+  const hasMultipleValues = Array.isArray(value) && value.length !== 1;
+
+  const { data, isLoading } = useGetParameterValuesQuery(
+    {
+      parameter: normalizeParameter(parameter),
+      field_ids: getNonVirtualFields(parameter).map(field => Number(field.id)),
+    },
+    {
+      skip:
+        !parameter ||
+        parameterHasNoDisplayValue(value) ||
+        Boolean(parameter.values_source_config?.values) ||
+        hasMultipleValues,
+    },
+  );
+
+  if (isLoading) {
+    return <span>{t`Loading…`}</span>;
```

Comment:
Is this product-approved design for the loading state?

We should add a test for it.

Reply:
I added tests for the rendering and loading state.

The product team says it's fine to fallback to rendering the value, so that's what I've implemented now.